Lesson_04/l04_h01_if_else.py

```diff
@@ -0,0 +1,39 @@
+"""
+HW:3[Volkov Anton].
+
+we have four values w,x,y,z
+write if-elif-else statement that will search minimum value
+and print smth aka "'y' is minimum value'
+advice use python debugger and different values to check your algorithm
+w, x, y, z = 100, 200, 40, 300
+"""
+w, x, y, z = 100, 200, 40, 300
+# 1. I've written my solution use python debugger to check my algorithm
```

Comment:
Thx for own algorithm ( I hope this is really your solution). Second solution looks as copy -paste from somewhere ) But not critical. I was focused on option 1.